scripts/tests.sh

```diff
@@ -21,7 +21,6 @@ if [[ -d "tests/" ]]; then
 
   if [[ -f "tests/requirements.txt" ]]; then
   pip install -r tests/requirements.txt
```

Comment:
don't we need a setup.py here?

Reply:
We shouldn't be using `setup.py` anymore.

See discussions in Slack.

1. Executors are not meant to be pypi packages
2. do relative imports in tests